### ISSUE TITLE
sphinx docs seems lost bullet points?

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#6313]
Looking at http://docs.datalad.org/en/latest/customization.html it seems we have lost bullet points for list items. (comapre to http://docs.datalad.org/en/stable/customization.html ).  I don't see something obvious in what in the diff from the latest release (which I guess based on 0.15.3)

```shell
$> git diff 0.15.3...origin/master --stat -- docs      
 docs/source/basics_cmdline.rst.in                  |   2 +-
 docs/source/cmdline.rst                            |  17 ++++++++++---
 docs/source/design/application_vs_library_mode.rst |  59 ++++++++++++++++++++++++++++++++++++++++++
 docs/source/design/batched_command.rst             |  48 +++++++++++++++++++++++++++++++++++
 docs/source/design/docstrings.rst                  |  83 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 docs/source/design/drop.rst                        |   1 +
 docs/source/design/file_url_handling.rst           |  86 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 docs/source/design/index.rst                       |   8 ++++++
 docs/source/design/pos_vs_kw_parameters.rst        |  51 +++++++++++++++++++++++++++++++++++++
 docs/source/design/result_records.rst              | 216 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 docs/source/design/standard_parameters.rst         |  48 +++++++++++++++++++++++++++++++++++
 docs/source/design/threaded_runner.rst             |  87 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 docs/source/modref.rst                             |   6 ++---
 13 files changed, 705 insertions(+), 7 deletions(-)
```

**Comment by adswa** [comment]
That's funny, if I build the docs locally on maint or master I get the bullet points. https://stackoverflow.com/questions/67542699/readthedocs-sphinx-not-rendering-bullet-list-from-rst-file and https://github.com/readthedocs/sphinx_rtd_theme/issues/1115 seems to suggest it could be due to docutils ``0.17``.

**Comment by adswa** [comment]
Fixed in #6346
